thirdparty/cmdconfig/commands/fnevalcmd.go

```diff
@@ -252,17 +196,8 @@ func toStorageMounts(mounts []string) []runtimeutil.StorageMount {
 	return sms
 }
 
-func (r *RunFnRunner) preRunE(c *cobra.Command, args []string) error {
-	if !r.EnableStar && (r.StarPath != "" || r.StarURL != "") {
-		return errors.Errorf("must specify --enable-star with --star-path and --star-url")
-	}
-
-	if !r.EnableExec && r.ExecPath != "" {
-		return errors.Errorf("must specify --enable-exec with --exec-path")
-	}
-
-	if c.ArgsLenAtDash() >= 0 && r.Image == "" &&
-		!(r.EnableStar && (r.StarPath != "" || r.StarURL != "")) && !(r.EnableExec && r.ExecPath != "") {
+func (r *EvalFnRunner) preRunE(c *cobra.Command, args []string) error {
+	if c.ArgsLenAtDash() >= 0 && r.Image == "" && r.ExecPath == "" {
 		return errors.Errorf("must specify --image")
```

Comment:
add `or --exec-path` to the error message.

Reply:
done